components/automate-ui/src/app/pages/+compliance/shared/reporting/stats.service.ts

```diff
@@ -215,15 +217,26 @@ export class StatsService {
   /* TODO Auth team: Helper functions that are only not private because
      they have unit testing. Should we delete their tests and make
      them private? */
-  getControlStatus(control): string {
-    const statuses = control.results.map(r => r.status);
-    const waived = control.waiver_data && control.waiver_data.skipped_due_to_waiver;
+  getControlStatus(control): ControlStatus {
+    // In some cases, nodes don't have results data.  In these
+    // cases we want to display those nodes as skipped
+    const hasResults = control.results.length > 0;
+    if (!hasResults) { return 'skipped'; }
```

Comment:
@vjeffrey - said guard clause.   just checking if that results array has anything in it.  if its empty - its skipped.

Reply:
ah thank you -- can we change the skipped to passed real quick? i double checked the backend and when no results we actually say passed (which is kinda weird to me but i guess we should get consistent before changing)

Reply:
@vjeffrey - no prob! updated to `passed`

Reply:
ahh jk - default is passed....updating now

Reply:
thank you!

Reply:
ok - default is set to passing now, also added a line checking for failed